manual_jb/content/analyses/tornadoepn_dmg.md

```diff
@@ -28,17 +28,16 @@ key name | type | name | description
 
 key name | type | name | description
 --- | --- | --- | ---
-`epn_node` <sup>*</sup> | `incore:epnNodeVer1` | EPN node | A dataset with nodes of the electric power network.
-`epn_link` <sup>*</sup> | `incore:epnLinkeVer1` | EPN link | A dataset with links of the electric power network.
+`epn_network` <sup>*</sup> | `incore:epnNetwork` | EPN network dataset | A network dataset of the electric power network.
 `seed` <sup>*</sup> | `int` | Seed | Initial value to seed the random number generator.
 `tornado` <sup>*</sup>| `incore:tornadoWindfield` | Tornado dataset | A tornado winds dataset.
 
 **Output Datasets** 
 
 key name | type | parent key | name | description
 --- | --- | --- | --- | ---
-`result` <sup>*</sup> | `incore:tornadoEPNDamageVer2` | `epn_node` | Results | A dataset containing results <br>(format: CSV).
-`damage_result` <sup>*</sup> | `incore:tornadoEPNDamageSupplement` | `epn_node` | Results | Information about applied hazard value and fragility<br>(format: JSON).
+`result` <sup>*</sup> | `incore:tornadoEPNDamageVer3` | `epn_network` | Results | A dataset containing results <br>(format: CSV).
+`metadata` <sup>*</sup> | `incore:tornadoEPNDamageSupplement` | | Results | Information about applied hazard value and fragility<br>(format: JSON).
```

Comment:
Can you re-add epn_network to the metadata table row? It looks like for all other analyses we made the parent key the same for both the damage CSV and the JSON, but it's not clear why since we can't combine the JSON with the shapefile like we do with the CSV results.